src/components/ResourceList/components/BulkActions/BulkActions.tsx

```diff
@@ -67,11 +67,11 @@ interface State {
 }
 
 const slideClasses = {
-  appear: classNames(styles.Slide, styles['Slide-appear']),
-  appearActive: classNames(styles.Slide, styles['Slide-appearing']),
-  enter: classNames(styles.Slide, styles['Slide-enter']),
-  enterActive: classNames(styles.Slide, styles['Slide-entering']),
-  exit: classNames(styles.Slide, styles['Slide-exit']),
+  appear: `${styles.Slide} ${styles['Slide-appear']}`,
+  appearActive: `${styles.Slide} ${styles['Slide-appearing']}`,
+  enter: `${styles.Slide} ${styles['Slide-enter']}`,
+  enterActive: `${styles.Slide} ${styles['Slide-entering']}`,
+  exit: `${styles.Slide} ${styles['Slide-exit']}`,
```

Comment:
Is using `classNames` impacting the tree-shakability?

Reply:
Whoops, that was me testing something. Didn't want to commit this.  I've reset this back to what it was originally.

But yes, usage of `classNames` isn't treeshakable because [rollup isn't treating the spread parameters as an array](https://github.com/rollup/rollup/issues/3690).

This isn't a big deal as it only affects cases that aren't inside a called function which is pretty minimal and hopefully rollup can be patched so it will just work.